tests/config/nextflow.config

```diff
@@ -6,7 +6,7 @@ params {
 
 process {
     cpus = 4
-    memory = 5.GB
+    memory = 8.GB
```

Comment:
Tests were not passing with 5.GB?

Reply:
Yes. These synth tools are very taxing. Could there be a way to fix specific processes resources ? Or must it be affected globally?

Reply:
I suggested a way in a comment on the `nextflow.config` file. Revert it here to 5.GB and try adding it to your config file.